packages/relay/src/formatters.ts

```diff
@@ -301,6 +301,14 @@ const getFunctionSelector = (data?: string): string => {
   return data.replace(/^0x/, '').substring(0, 8);
 };
 
+const tinybarsToWeibars = (value: number | null) => {
```

Comment:
I know this addresses this comment https://github.com/hashgraph/hedera-json-rpc-relay/pull/3057#discussion_r1791757104
Not sure if its relevant for a util function

Reply:
I think it's fine for now.
We can relax in a follow up PR if you think it's overkill.